project/AkkaBuild.scala

```diff
@@ -100,6 +100,7 @@ object AkkaBuild {
         "UTF-8",
         "-feature",
         "-unchecked",
+        "-deprecation",
```

Comment:
Do I understand it correctly that the `nowarn` annotations are added because of this?
There is a risk that we later don't notice that we use deprecated APIs, but I guess that's a risk we have to take while cross compiling with so many Scala versions.

Reply:
noop, before with this, it field without any messages, add this to see where need to adjust.